lib/include/hse_ikvdb/kvdb_rparams.h

```diff
@@ -75,10 +75,11 @@ struct kvdb_rparams {
 
     uint32_t dur_bufsz_mb;
     uint32_t dur_intvl_ms;
-    uint8_t  dur_throttle_lo_th;
-    uint8_t  dur_throttle_hi_th;
+    uint32_t dur_sz_bytes;
     bool     dur_enable;
     bool     dur_buf_managed;
+    uint8_t  dur_throttle_lo_th;
+    uint8_t  dur_throttle_hi_th;
```

Comment:
what does _th mean?

Reply:
I think "threshold", not sure..

Reply:
We should figure it out :-)  Seems to me dur_throttle_lwm and ru_throttle_hwm would be more clear, if that's what they are (watermarks that is...)

Reply:
Will address this in a separate cleanup.